.github/workflows/run-builds.yml

```diff
@@ -0,0 +1,70 @@
+name: Run Builds
+
+on:
+  schedule:
+    - cron: '*/20 * * * *' # Run every 20 minutes
+  pull_request:
+    branches: [main]
```

Comment:
Once tests pass, let's remove these lines and only run on cron.

```suggestion
```

Reply:
ok!